src/main/java/com/google/sps/servlets/ListClassesServlet.java

```diff
@@ -0,0 +1,30 @@
+package com.google.sps.servlets;
+
+import com.google.sps.Course;
+import java.io.IOException;
+import java.util.*;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/homepage")
+public class ListClassesServlet extends HttpServlet{
+
+    ArrayList<Course> courseList = new ArrayList<>();
```

Comment:
Initialize the array here. You can use `Arrays.asList` or `List.of(...)`. See https://www.geeksforgeeks.org/initialize-an-arraylist-in-java/